src/Elcodi/Bundle/CartBundle/Resources/config/eventListeners.yml

```diff
@@ -11,7 +11,7 @@ services:
             - @elcodi.cart.manager
             - @elcodi.wrapper.currency
             - @elcodi.converter.currency
-            - @elcodi.manager.configuration
+            - "@=elcodi_config('product.use_stock', false)"
```

Comment:
Ok, I stand corrected, this PR add also support for default values. :clap::clap::clap: